src/oc/storage/api/entries.clj

```diff
@@ -396,6 +415,64 @@
       (timbre/error "Failed removing bookmark for entry:" entry-for "and user" (-> ctx :user :user-id))
       false)))
 
+;; Polls
```

Comment:
@bago2k4 I wonder if polls just shouldn't get its own namespace. It has routes, resources, helper fn's etc.